### ISSUE TITLE
Has the support for decko been implemented yet?

### ISSUE THREAD

**Issue by xmstspider** [ytdl-org/youtube-dl#23471]
## Checklist

- [X] I'm asking a question
- [X] I've looked through the README and FAQ for similar questions
- [X ] I've searched the bugtracker for similar questions including closed ones

## Question

Hi,

I tried to download from https://decko.ceskatelevize.cz/ but youtube-dl says the URL is not supported.

I found #12713 and #12719 but the mentioned changes are not part of the code nor the release.

What is the status of this feature.

My youtube-dl version is 2019.09.28-1 but I did not find the support for decko either when I searched the code here on github.

Thanks

**Comment by mcepl** [comment]
Nope, I still cannot download https://decko.ceskatelevize.cz/video/e21551313003 .
